dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/services/settings/SettingsStore.java

```diff
@@ -103,23 +106,45 @@ public void setFilePropertiesSettings(final boolean settings) {
         filePropertiesSettings.overwrite(settings);
     }
 
+    public ShowCachedJobSettings getShowCachedJobSettings() {
+        return showCachedJobSettings;
+    }
+
+    public void setShowCachedJobSettings(final boolean settings) {
+        dirty = true;
+        showCachedJobSettings.overwrite(settings);
+    }
+
+
     //method to include new json entry to settings.json file if any new setting property is introduced.
     public static void writeNewSettingsToSettingsJsonFile() {
         try (final Stream<String> stream = Files.lines(PATH)) {
-            stream.forEach(e -> {
-                if (!e.contains("filePropertiesSettings")) {
-                    StringBuilder newFile = new StringBuilder(e);
+            stream.forEach(entry -> {
+                if (!entry.contains("filePropertiesSettings")) {
+                    StringBuilder newFile = new StringBuilder(entry);
                     newFile.deleteCharAt(newFile.length() - 1);
                     newFile = newFile.append(",\"filePropertiesSettings\":{\"filePropertiesEnable\":true}}");//file property is true by default..adding this new setting to the file
                     try (final BufferedWriter writer = Files.newBufferedWriter(PATH)) {
                         writer.write(newFile.toString());
-                    } catch (final IOException ex) {
-                        LOG.error("Failed to save settings", ex);
+                    } catch (final IOException e) {
+                        LOG.error("Failed to save settings", e);
+                    }
+                }
+                if (!entry.contains("showCachedJobSettings")) {
+                    StringBuilder newFile = new StringBuilder(entry);
```

Comment:
Does this value have to be reassigned on line 136?  If it doesn't then this should be made final.